platform/darwin/docs/theme/assets/css/jazzy.css.scss

```diff
@@ -488,9 +536,16 @@ pre code {
 }
 
 .language {
+  background: $color_light;
```

Comment:
Subtly color-coding the two languages is a good idea. I’ll have to consult with you when I add AppleScript code snippets to the macOS SDK documentation. 😉 On the other hand, perhaps we should take this opportunity to add a language toggle to make this color-coding unnecessary: #7448.